lib/config.js

```diff
@@ -15,11 +15,12 @@ var path = require('path');
 /**
  * Get configuration for a project, falling back to default configuration if
  * project does not provide a configuration of its own
- * @param {String} root - root path of project
+ * @param {Object} project - The Project model tied to the package.json
  * @returns {Configuration} configuration to use for project
  */
-function config(root) {
-  var configFile = path.join(root, 'config', 'coverage.js');
+function config(project) {
```

Comment:
I think this makes sense because I could still keep the logic to construct the path to `coverage.js` inside `config` like @rwjblue suggested but also not take in the whole `project` object. 

I can make this change if there are no objections.